Dangerfile

```diff
@@ -43,10 +43,13 @@ end
 
 forgot_tests = !git.modified_files.include?("src/tests/tests.py")
 
-if forgot_tests
-	warn('It appears that you forgot to add a Unit Test to the test file.\n Please add a test and upload the new version.\n The test file can currently be found at: ./src/tests/tests.py\n If this is a "trivial" change add #trivial to the title')
+
+if forgot_tests and not declared_trivial
+	warn("It appears that you forgot to add a Unit Test to the test file.\r\n Please add a test and upload the new version.\r\n The test file can currently be found at: ./src/tests/tests.py\r\n If this is a 'trivial' change add #trivial to the title")
+
 else
-	message("Thanks for remembering to add a test!")
+      msg = not forgot_tests ? "Thanks for remembering to add a test!" : "Thanks for remembering to declare trivial!"
+     message(msg)
 end
```

Comment:
Try using ! instead of not here. I think that should fix it if it was causing a problem before.